nomad/grapl-core.nomad

```diff
@@ -1200,6 +1205,7 @@ job "grapl-core" {
         PLUGIN_REGISTRY_DB_USERNAME      = var.plugin_registry_db_username
         PLUGIN_BOOTSTRAP_CONTAINER_IMAGE = var.container_images["plugin-bootstrap"]
         KERNEL_ARTIFACT_URL              = var.plugin_registry_kernel_artifact_url
+        ROOTFS_ARTIFACT_URL              = var.plugin_registry_rootfs_artifact_url
```

Comment:
I took an executive decision to stop prepending PLUGIN_BOOTSTRAP_ at this level; feel free to say 'i don't like that'. 
But especially with our structopts it feels really weird in plugin-registry code to reference `config.plugin_registry_kernel_artifact_url` when you already _know_ you're in plugin_registry land.

Reply:
I think there are two related but distinct concerns here.

When you're in the code of the plugin registry service, I agree that constantly typing things like `config.plugin_registry_kernel_artifact_url` is a downer. You're really _doubly_ namespaced, in a way: you're in the plugin registry crate, and you're also dealing with a type named `PluginRegistryServiceConfig`. So yeah, just call that `config.kernel_artifact_url` and be done with it; no need to add another layer of namespacing.

Environment variables are slightly different, though, since they really don't _have_ any kind of native namespacing method, apart from naming conventions. It can be useful to have all the environment variables for configuring a service to be clearly named for that service. It may just be a holdover from pre-cloud, pre-container days when multiple services could be on the same machine, and you wanted to ensure that values intended for one service didn't accidentally make their way into another service (just which service is supposed to get `DB_USERNAME`, for instance?). Be that as it may, I (personally) still find it useful from an organizational / documentation perspective to name environment variables in this way.

So, my two cents is: keep the `PLUGIN_REGISTRY_` prefix on the environment variables, but ditch it in the Rust configuration code.